packages/client/components/ArchiveTaskRoot.tsx

```diff
@@ -31,7 +32,7 @@ const ArchiveTaskRoot = ({teamId, team, showHeader}: ArchiveTaskRootProps) => {
     <QueryRenderer
       environment={atmosphere}
       query={query}
-      variables={{teamId, first: 40}}
+      variables={{teamId, first: 40, includeTeamMembers}}
```

Comment:
`includeTeamMembers` works for this instance, but it's going to break when have multiple filters.

how about the following:
- variables include `teamIds, userIds, first, after`
- if `teamIds` is empty, then do not filter out by teamId
- if `userIds` is empty, then do not filter out by userId

If we do this, then the following is possible:
- Get the Team Archive: `teamIds: ['team123], userIds: []`
- Get the User Archive for the Viewer: `teamIds: [], userIds: ['bruce']`
- Get the User Archive for someone else: `teamIds: [], userIds: ['matt']`

Then, we can do more advanced things. like let's say Matt wants to see all of Bruce's tasks, but Bruce is on a team that Matt isn't on, so Matt shouldn't see those tasks.
- teamIds: [], userIds: ['bruce']
to make sure Matt doesn't see the private stuff:
```ts
const mattsTeamMembers = await dataLoader.get('teamMembersByUserId').load('matt')
const teamIds = mattsTeamMembers.map(({teamId}) => teamId)
const brucesTasksVisibleByMatt = r.table('Task').getAll(r.args('bruce'), {index: 'userId'}).filter((task) => r(teamIds).contains(task('teamId'))).filter((task) => task('tags').contains('private').not())
```